src/satosa/backends/apple.py

```diff
@@ -113,11 +113,10 @@ def response_endpoint(self, context, *args):
 
         # convert "string or Boolean" claims to actual booleans
         for bool_claim_name in ["email_verified", "is_private_email"]:
-            userinfo[bool_claim_name] = (
-                True
-                if userinfo[bool_claim_name] == "true"
-                else False
-            )
+            if type(all_user_claims.get(bool_claim_name)) == str:
```

Comment:
do we care if it is a str,
or whether the claim is there?

```suggestion
            if bool_claim_name in all_user_claims.keys():
```

Reply:
Yes, because the claim also might be a boolean, according to Apple's spec, and it there was `True` as boolean, your code woud turn it into `False`, because `False != "true"`